### PR TITLE
0.25.4 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: cloud/stable/07_resources/01_release-notes.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ description: "Astronomer Cloud Release Notes."
 
 ## Astronomer v0.25
 
+### v0.25.4
+
+Release Date: June 31st, 2021
+
+#### Bug fixes
+
+- Fixed an issue where Workspace Admin service accounts could not log in to Astronomer.
```
Comment (bote795) [inline cloud/stable/07_resources/01_release-notes.md:15]: could not CRUD (didn't have access)

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 2]
```diff
@@ -12,6 +12,18 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
...
+
+Release Date: June 31st, 2021
+
+### Minor Improvements
+
+- Added support for installing Astronomer via ArgoCD, which facilitates continuous delivery for Kubernetes applications. For more information, read the [Astronomer Forum](https://forum.astronomer.io/t/installing-astronomer-with-argocd/1341).
+
+### Bug fixes
+
+- Fixed an issue where Workspace Admin service accounts could not log in to Astronomer.
```
Comment (bote795) [inline enterprise/v0.25/10_resources/01_release-notes.md:25]: same as above

File: cloud/stable/07_resources/01_release-notes.md [hunk 3]
```diff
@@ -6,6 +6,14 @@ description: "Astronomer Cloud Release Notes."
 
 ## Astronomer v0.25
 
+### v0.25.4
+
+Release Date: June 31st, 2021
```
Comment (paolaperaza) [inline cloud/stable/07_resources/01_release-notes.md:11]: Should this be today (July 2nd, 2021)?

File: cloud/stable/07_resources/01_release-notes.md (hunk 1 above)
Comment (paolaperaza) [inline cloud/stable/07_resources/01_release-notes.md:15]:
```suggestion
- Fixed an issue where Workspace Service Accounts lost access to Astronomer.
```

(Don't think it was a problem exclusive to Admin - I think it was all Workspace SAs)

File: enterprise/v0.25/10_resources/01_release-notes.md (hunk 2 above)
Comment (paolaperaza) [inline enterprise/v0.25/10_resources/01_release-notes.md:25]:
```suggestion
- Fixed an issue where Workspace Service Accounts lost access to Astronomer.
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 4]
```diff
@@ -12,6 +12,18 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.4
+
+Release Date: June 31st, 2021
+
+### Minor Improvements
+
+- Added support for installing Astronomer via ArgoCD, which facilitates continuous delivery for Kubernetes applications. For more information, read the [Astronomer Forum](https://forum.astronomer.io/t/installing-astronomer-with-argocd/1341).
```
Comment (paolaperaza) [inline enterprise/v0.25/10_resources/01_release-notes.md:21]:
```suggestion
- Added support for installing Astronomer Enterprise via ArgoCD, which facilitates continuous delivery for Kubernetes applications. For more information, read the [Astronomer Forum](https://forum.astronomer.io/t/installing-astronomer-with-argocd/1341).
```

File: enterprise/v0.25/10_resources/01_release-notes.md [hunk 5]
```diff
@@ -12,6 +12,18 @@ If you're looking to upgrade to Astronomer v0.25, refer to [Upgrade to Astronome
 
 We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## v0.25.4
+
+Release Date: June 31st, 2021
```
Comment (paolaperaza) [inline enterprise/v0.25/10_resources/01_release-notes.md:17]:
```suggestion
Release Date: July 2nd, 2021
```

File: cloud/stable/07_resources/01_release-notes.md (hunk 3 above)
Comment (paolaperaza) [inline cloud/stable/07_resources/01_release-notes.md:11]:
```suggestion
Release Date: July 2nd, 2021
```

**Review by paolaperaza — APPROVED** [review]
Some minor copy suggestions but LGTM @jwitz

**Comment by vishwas-astro** [comment]
LGTM after @bote795 and @paolaperaza's comments are addressed.

**Comment by vishwas-astro** [comment]
@jwitz - We have not released 0.25.4 yet, Was not expecting you to merge it right away

**Pull request merged**